bigchaindb/commands/bigchain.py

```diff
@@ -146,7 +147,12 @@ def run_init(args):
 def run_drop(args):
     """Drop the database"""
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
-    db.drop(assume_yes=args.yes)
+    try:
+        db.drop(assume_yes=args.yes)
+    except DatabaseDoesNotExist:
+        db_name = db.get_database_name()
+        print('Cannot drop \'{name}\'. The database does not exist.'.format(name=db_name), file=sys.stderr)
```

Comment:
[Double quotes](https://github.com/bigchaindb/bigchaindb/blob/master/PYTHON_STYLE_GUIDE.md#single-or-double-quotes) are allowed in this context.